### DISCUSSION TITLE
Text Decals

### DISCUSSION THREAD

**Discussion by TIKIRobo** [tsunamayo/Starship-EVO#4067] [Suggestions]
I know that things like button labels and small text screens have been suggested in the past but it would also be nice to see text decals, sure we can make our own text but often that text looks ugly or has to be over a certain size because decals can only get so small 
![image](https://user-images.githubusercontent.com/56371294/126576028-041d99aa-6112-47e4-a3ba-76f2f3629fb4.png)
it would be nice to see the alphabet and 0-9 in decal form with scaling from 1/8 to 32 with a similar/the same font as above 

also as a small side note it would be nice to see the 1/8 decals and blocks be able to be placed on the half and quarter grid, sure we can offset the blocks but that still leaves the decals locked to the 1/8 grid

**Comment by KINGrimson** [comment]
yes please, with the new 1/8 stickers i have been wanting to make new numbers for floor indication, before they where to big, but with the 1/8 a few can be done but zome just dont look like like a 4 at the zmallest you can do it,  it just doesnt look right, just having the ofzet on the 1/8 would be great
![unknown](https://user-images.githubusercontent.com/74992043/126577005-acb1035e-bc6b-4989-9bdc-db6849153f00.png)

**Reply by KINGrimson** [reply]
![unknown-150](https://user-images.githubusercontent.com/74992043/126577053-e8218572-6118-48cb-9974-4e0a14e8d835.png)
better picture
